src/outputs/harbor_output.ml

```diff
@@ -395,9 +406,8 @@ module Make (T : T) = struct
 
       val mutable chunk_len = 0
 
-      val mutable burst_data = []
-
-      val mutable burst_pos = 0
+      val burst_data =
+        Strings.Mutable.of_strings Strings.empty
```

Comment:
```suggestion
        Strings.Mutable.empty ()
```

Reply:
Yes, why didn't you commit this change?

Reply:
I'm waiting for more to commit all of them in a batch..